src/EnergyPlus/DXCoils.cc

```diff
@@ -4095,13 +4095,17 @@ namespace DXCoils {
 				if ( DXCoil( DXCoilNum ).FuelType != FuelTypeElectricity ) {
 					if ( DXCoil( DXCoilNum ).MSWasteHeat( I ) == 0 ) {
 						if ( lAlphaBlanks( 18 + ( I - 1 ) * 6 ) ) {
-							ShowSevereError( RoutineName + CurrentModuleObject + "=\"" + DXCoil( DXCoilNum ).Name + "\", missing" );
-							ShowContinueError( "...required " + cAlphaFields( 18 + ( I - 1 ) * 6 ) + " is blank." );
+							ShowWarningError( RoutineName + CurrentModuleObject + "=\"" + DXCoil( DXCoilNum ).Name + "\", missing" );
+							ShowContinueError( "... " + cAlphaFields( 18 + ( I - 1 ) * 6 ) + " is blank." );
 						} else {
-							ShowSevereError( RoutineName + CurrentModuleObject + "=\"" + DXCoil( DXCoilNum ).Name + "\", invalid" );
+							ShowWarningError( RoutineName + CurrentModuleObject + "=\"" + DXCoil( DXCoilNum ).Name + "\", invalid" );
 							ShowContinueError( "...not found " + cAlphaFields( 18 + ( I - 1 ) * 6 ) + "=\"" + Alphas( 18 + ( I - 1 ) * 6 ) + "\"." );
 						}
+<<<<<<< .mine
+//						ErrorsFound = true;
+=======
```

Comment:
@lgu1234 The builds are failing due to these conflict markers <<<< .mine and below >>>> .theirs.  Also, the end of the unit test file needs a newline.